### ISSUE TITLE
no name in kde plasma system tray entries settings

### ISSUE THREAD

**Issue by ilobilo** [Vencord/Vesktop#689]
### Content

as seen in the screenshot below, vesktop has no name assigned in kde plasma system tray settings. this causes it to interfere with other programs that also don't have a name set. (for example unity)
![image](https://github.com/Vencord/Vesktop/assets/68286835/18ecd281-3914-444c-ac20-1460d0224d7a)

**Comment by PeterStrick** [comment]
> ### Content
> as seen in the screenshot below, vesktop has no name assigned in kde plasma system tray settings. this causes it to interfere with other programs that also don't have a name set. (for example unity) ![image](https://private-user-images.githubusercontent.com/68286835/341841911-18ecd281-3914-444c-ac20-1460d0224d7a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg5OTIzOTcsIm5iZiI6MTcxODk5MjA5NywicGF0aCI6Ii82ODI4NjgzNS8zNDE4NDE5MTEtMThlY2QyODEtMzkxNC00NDRjLWFjMjAtMTQ2MGQwMjI0ZDdhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjIxVDE3NDgxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgxMDkwN2VlODkyOTlmMDFiOGFhNWFlNzk4NDA2ODgxYmZlZGQxMjg0M2E2Yjk4YzI0MWJlMWRhMWNlMGE3MzUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.oigOrKmQCqLd96j0xA7Iqu8jM6kAzR_BXg-czWbE5PY)

This also happens with Signal Desktop, as it also doesn't have a name, and even the normal Discord Application
![image](https://github.com/Vencord/Vesktop/assets/60312421/53b2c12f-7c84-4f61-b8f0-f0f51c3b44a8)

**Comment by Vendicated** [comment]
i don't think we can fix this. at least i don't know one would specify the name. the [electron tray api](https://www.electronjs.org/docs/latest/api/tray) has no such options

**Comment by Vendicated** [comment]
blocked by https://github.com/electron/electron/issues/40936
